### ISSUE TITLE
Getting this Error using the Instructions in the Readme

### ISSUE THREAD

**Issue by surfer190** [Riari/laravel-forum#75]
```
There is no column with name 'category_id' on table 'forum_categories'
```

**Comment by surfer190** [comment]
```
Laravel will implicitly put any commands that add new or modify existing columns at the very beginning of the commands array when the migration is executed
```

Are there tests for these migrations?

**Comment by Riari** [comment]
Unfortunately no, but it's a simple fix at least. Thanks for the heads up.

Fix is in 361115b591e0dd896e57a7c644f5a66cb08ebbe1
